.markdownlint.json

```diff
@@ -10,5 +10,6 @@
     "MD036": false,
     "MD046": {
         "style": "fenced"
-    }
+    },
+    "DOCSMD010": false
```

Comment:
This suppresses the linter warning on `[foo](xref:bar#baz)` - the linker doesn't like the hash sign and suggests %23 instead, but that doesn't get picked up properly by the doc system...

Reply:
Ran into this yesterday.